src/models/propulsion/FGTurbine.cpp

```diff
@@ -220,7 +220,11 @@ double FGTurbine::Run()
   OilTemp_degK = Seek(&OilTemp_degK, 366.0, 1.2, 0.1);
 
   if (!Augmentation) {
-    correctedTSFC = TSFC->GetValue() * sqrt(T/389.7) * (0.84 + (1-N2norm)*(1-N2norm));
+    // If TSFC function is supplied don't perform any correction/denormalisation
+    if (dynamic_cast<FGFunction*>(TSFC))
+      correctedTSFC = TSFC->GetValue();
+    else
+      correctedTSFC = TSFC->GetValue() * sqrt(T/389.7) * (0.84 + (1-N2norm)*(1-N2norm));
```

Comment:
I'd rather create a new class `FGTSFC` to specialize `GetValue()` with the expression above. Similar to what I did for `FGSpoolUp`
https://github.com/JSBSim-Team/jsbsim/blob/a00129aa73c87d5322ca086aa9ecbf26c0be4cc5/src/models/propulsion/FGTurbine.h#L325-L340
The point being that I think the fuel flow management at idle will need to move to that class as well.

Reply:
Just pushed a commit to go with the suggested approach.